src/index.js

```diff
@@ -270,6 +270,23 @@ Spanner.int = function(value) {
   return new codec.Int(value);
 };
 
+/**
+ * Helper function to get a Cloud Spanner Struct object.
+ *
+ * @param {object} value The struct as a JSON object.
+ * @returns {object}
+ *
+ * @example
+ * const Spanner = require('@google-cloud/spanner');
+ * const struct = Spanner.struct({
+ *   user: 'bob',
+ *   age: 32
+ * });
+ */
+Spanner.struct = function(value) {
```

Comment:
why can't users directly pass in a json object as the value of struct parameter? Why do they have to wrap it in `Spanner.struct`

Reply:
Any updates?

Reply:
I suppose we could allow them to pass objects directly, my thinking was that by default we don't give the users structs in the form of plain old objects by default, so accepting them by default might be confusing.

@stephenplusplus had brought up the same point and I thought it might be better to allow them to pass in an object by default once we defaulted to returning json.